core/src/scrim/scrim.mod.resolver.ts

```diff
@@ -79,8 +79,8 @@ export class ScrimModuleResolver {
             nullable: true,
         }) status?: ScrimStatus,
     ): Promise<Scrim[]> {
-        if (!user.currentOrganizationId) throw new GraphQLError("User is not connected to an organiazation");
-
+        if (!user.currentOrganizationId) throw new GraphQLError("Player is not connected to an organization");
```

Comment:
`Player has not selected an organization` may be a better way to say this; not sure.

Reply:
It's a bit of a conundrum anyway, since we don't offer a user a way to select an organization atm.